packages/env/lib/env.js

```diff
@@ -363,3 +371,22 @@ async function resetDatabase( environment, { dockerComposeConfigPath } ) {
 
 	await Promise.all( tasks );
 }
+
+/**
+ * Sets the correct user and permissions on the wp-conf.php file.
+ *
+ * @param {string} wordPressPath The path to the WordPress source code.
```

Comment:
Wouldn't `coreSourcePath` be more consistent?

Reply:
Sure. :)